purchase_console/models/requisition.py

```diff
@@ -84,12 +84,33 @@ def _inverse_suppliers(self):
                                                " do you have of stock.")
 
     @api.multi
+    def confirm_call(self):
+        self.ensure_one()
```

Comment:
Is really need the `self.ensure_one()` here?

FYI we can use `self.write` for singleton and non-singleton records